docs/tasks/access-kubernetes-api/setup-extension-api-server.md

```diff
@@ -0,0 +1,53 @@
+---
+title: Setup an extension API server
+assignees:
+- lavalamp
+- cheftako
+- chenopis
+---
+
+{% capture overview %}
+
+Setting up an extension API server to work the aggregation layer allows the Kubernetes apiserver to be extended with additional APIs, which are not part of the core Kubernetes APIs. 
+
+{% endcapture %}
+
+{% capture prerequisites %}
+
+* You need to have a Kubernetes cluster running.
+* You must [configure the aggregation layer](/docs/tasks/configure-aggregation-layer) and enable the apiserver flags.
+
+{% endcapture %}
+
+{% capture steps %}
+
+## Setup an extension api-server to work with the aggregation layer
+
+If you use an existing 3rd party solution, such as [apiserver-builder](https://github.com/Kubernetes-incubator/apiserver-builder/blob/master/README.md), it should generate a skeleton and perform all of the following steps. Alternatively, you can also look at the [sample-apiserver](https://github.com/kubernetes/sample-apiserver/blob/master/README.md) under Kubernetes for ways the following can be done.
```

Comment:
I was a bit confused by this blurb at first (e.g. why the steps below lacked detail). Consider the following wording for more clarity?

> The following steps describe how to set up an extension-apiserver *at a high level*. For a concrete example of how they can be implemented, you can look at the [sample-apiserver](https://github.com/kubernetes/sample-apiserver/blob/master/README.md) in the Kubernetes repo.
>
> Alternatively, you can use an existing 3rd party solution, such as [apiserver-builder](https://github.com/Kubernetes-incubator/apiserver-builder/blob/master/README.md), which should generate a skeleton and automate all of the following steps for you.